### PR TITLE
Misc. UI Polish

### LINKED ISSUE #12064: Make Variables empty state consistent with other resource lists
- Use `ComponentSize.Large` for size of empty state
- Highlight word "Variables"

variables-empty-state

### LINKED ISSUE #12065: Fix Tasks header
Header halves are misaligned when rendered inside the org page

screen shot 2019-02-21 at 10 43 22 am

### LINKED ISSUE #12075: Remove view button, link using telegraf name instead
screen shot 2019-02-21 at 2 19 31 pm

### LINKED ISSUE #12077: Change "Setup Details" button to "Setup Instructions"
screen shot 2019-02-21 at 2 19 28 pm

This way it will match the name of the overlay

### LINKED ISSUE #12092: modal issue with creating an org
image

### LINKED ISSUE #12110: Buckets tab not highlighting when active
screen shot 2019-02-22 at 9 54 35 am

### PR DESCRIPTION
**Pull request by alexpaxton**
screen shot 2019-02-22 at 10 44 22 am
Closes #12064 

screen shot 2019-02-22 at 10 42 00 am
Closes #12110 

screen shot 2019-02-22 at 10 42 07 am
Closes #12075 
Closes #12077 

screen shot 2019-02-22 at 10 42 16 am
Closes #12065 

org-name-error-messages
Closes #12092 

  - [x] Rebased/mergeable
  - [x] Tests pass
  - [ ] http/swagger.yml updated (if modified Go structs or API)
  - [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by alexpaxton** [comment]
I really like how the Dashboards index can be displayed in 2 contexts and each context has its own header component. I wasn't sure if creating 2 different tasks headers was a good idea or not, so instead I have a conditional to render it as a `PageHeader` or a `TabContentsHeader`.

File: ui/src/organizations/components/CollectorList.tsx [hunk 1]
```diff
@@ -28,7 +28,8 @@ export default class CollectorList extends PureComponent<Props> {
         <IndexList>
           <IndexList.Header>
             <IndexList.HeaderCell columnName="Name" width="50%" />
-            <IndexList.HeaderCell columnName="Bucket" width="50%" />
+            <IndexList.HeaderCell columnName="Bucket" width="25%" />
+            <IndexList.HeaderCell columnName="" width="25%" />
```
Comment (OfTheDelmer) [inline ui/src/organizations/components/CollectorList.tsx]: what's this column?

File: ui/src/organizations/components/CollectorRow.tsx [hunk 2]
```diff
@@ -61,13 +62,7 @@ export default class CollectorRow extends PureComponent<Props> {
...
                 size={ComponentSize.ExtraSmall}
                 color={ComponentColor.Secondary}
-                text={'View'}
-                onClick={this.handleOpenConfig}
-              />
-              <Button
-                size={ComponentSize.ExtraSmall}
-                color={ComponentColor.Secondary}
-                text={'Setup Details'}
+                text={'Setup Instructions'}
```
Comment (OfTheDelmer) [inline ui/src/organizations/components/CollectorRow.tsx]:
```suggestion
                text="Setup Instructions"
```

File: ui/src/organizations/components/CreateOrgOverlay.tsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // Libraries
 import React, {PureComponent, ChangeEvent} from 'react'
+import _ from 'lodash'
```
Comment (OfTheDelmer) [inline ui/src/organizations/components/CreateOrgOverlay.tsx]: I've been seeing more of the `import {sample} from 'lodash'` style and the default import. I don't know which we prefer.

File: ui/src/organizations/components/CreateOrgOverlay.tsx [hunk 4]
```diff
@@ -108,4 +117,17 @@ export default class CreateOrgOverlay extends PureComponent<Props, State> {
       errorMessage: '',
     })
   }
+
+  private randomErrorMessage = (key: string): string => {
```
Comment (OfTheDelmer) [inline ui/src/organizations/components/CreateOrgOverlay.tsx]:
```suggestion
  private randomErrorMessage(key: string): string {
```

File: ui/src/organizations/components/OrgTasksPage.tsx [hunk 5]
```diff
@@ -95,47 +93,37 @@ class OrgTasksPage extends PureComponent<Props, State> {
...
-                onRemoveTaskLabels={onRemoveTaskLabels}
-                onUpdate={this.handleUpdateTask}
-                onRunTask={onRunTask}
-              />
-            )}
-          </FilterList>
-        </Page>
+        <TasksHeader
+          onCreateTask={this.handleCreateTask}
+          setSearchTerm={setSearchTerm}
```
Comment (OfTheDelmer) [inline ui/src/organizations/components/OrgTasksPage.tsx]: Should the searchWidget just be connected. I know in the following labels click update the searchTerm also has to be supplied back to the search widget so it reflects the most recent state of `searchTerm`.

Comment (OfTheDelmer) [reply ui/src/organizations/components/OrgTasksPage.tsx]: This is also the only `searchTerm` connected to the redux store and the rest aren't? Should `SearchWidget` be utilized in those org tabs also?

File: ui/src/organizations/components/OrgTasksPage.tsx [hunk 6]
```diff
@@ -95,47 +93,37 @@ class OrgTasksPage extends PureComponent<Props, State> {
...
+        <FilterList<Task>
+          searchTerm={searchTerm}
+          searchKeys={['name', 'labels[].name']}
+          list={this.filteredTasks}
+        >
+          {ts => (
+            <TasksList
+              searchTerm={searchTerm}
+              tasks={ts}
+              totalCount={this.totalTaskCount}
```
Comment (OfTheDelmer) [inline ui/src/organizations/components/OrgTasksPage.tsx]: Should this update to reflect ts.length?

```suggestion
              totalCount={ts.length}
```

File: ui/src/organizations/containers/OrgBucketsIndex.tsx [hunk 7]
```diff
@@ -60,7 +60,7 @@ class OrgBucketsIndex extends Component<Props> {
         <Page.Contents fullWidth={false} scrollable={true}>
           <div className="col-xs-12">
             <Tabs>
-              <OrganizationNavigation tab={'bucket_tab'} orgID={org.id} />
+              <OrganizationNavigation tab={'buckets_tab'} orgID={org.id} />
```
Comment (OfTheDelmer) [inline ui/src/organizations/containers/OrgBucketsIndex.tsx]:
```suggestion
              <OrganizationNavigation tab="buckets_tab" orgID={org.id} />
```

File: ui/src/tasks/components/TasksHeader.tsx [hunk 8]
```diff
@@ -17,16 +18,16 @@ interface Props {
   showInactive: boolean
   toggleOverlay: () => void
   showOrgDropdown?: boolean
-  showFilter?: boolean
+  isFullPage?: boolean
```
Comment (OfTheDelmer) [inline ui/src/tasks/components/TasksHeader.tsx]: Could be a good opportunity for an enum

File: ui/src/organizations/components/CollectorList.tsx (hunk 1 above)
Comment (alexpaxton) [reply ui/src/organizations/components/CollectorList.tsx]: The rows have a unnamed third column (which is usually reserved for actions), just making sure there are equal number of columns in the header as the actual table

File: ui/src/organizations/components/CollectorRow.tsx (hunk 2 above)
Comment (alexpaxton) [reply ui/src/organizations/components/CollectorRow.tsx]: I posted in #chronograf-code about this, I could go either way on the pattern

File: ui/src/organizations/components/CreateOrgOverlay.tsx (hunk 4 above)
Comment (alexpaxton) [reply ui/src/organizations/components/CreateOrgOverlay.tsx]: I suppose this function doesn't need the binding that comes from using an arrow function

File: ui/src/organizations/components/OrgTasksPage.tsx (hunk 6 above)
Comment (alexpaxton) [reply ui/src/organizations/components/OrgTasksPage.tsx]: Hmm good question

File: ui/src/organizations/components/CollectorRow.tsx (hunk 2 above)
Comment (OfTheDelmer) [reply ui/src/organizations/components/CollectorRow.tsx]: ahh okay, I missed that. will review

File: ui/src/organizations/components/CollectorList.tsx (hunk 1 above)
Comment (OfTheDelmer) [reply ui/src/organizations/components/CollectorList.tsx]: 👍

File: ui/src/tasks/components/TasksHeader.tsx (hunk 8 above)
Comment (alexpaxton) [reply ui/src/tasks/components/TasksHeader.tsx]: Having to use a prop for this does feel kind of weird. There has to be a better way for displaying the same header in two places with different wrapper components

Comment (OfTheDelmer) [reply ui/src/tasks/components/TasksHeader.tsx]: Yeah, we can just keep it in mind as we patterns emerge in the orgs pages.

**Pull request merged**